pymc3/tests/test_distributions_random.py

```diff
@@ -817,7 +816,6 @@ def ref_rand(size, lower, upper):
             pm.DiscreteUniform, {"lower": -NatSmall, "upper": NatSmall}, ref_rand=ref_rand
         )
 
-    @pytest.mark.xfail(reason="This distribution has not been refactored for v4")
```

Comment:
This one is not really doing an independent test, so I am not sure about the usefulness of it. Something like what is done in #4608 sounds less wasteful if all we want to check is that the right parameters are being used. Opinions?

Reply:
Will take care of this via #4608